honeycomb_tools/__main__.py

```diff
@@ -84,7 +84,7 @@ def prepare_videos_for_environment_for_day(
               help='start time of video to load expects format to be YYYY-MM-DDTHH:MM', required=True)
 @click.option('--end', help='end time of video to load expects format to be YYYY-MM-DDTHH:MM', required=True)
 def list_datapoints_for_environment_for_time_range(
-        ctx, environment_name, output_path, output_name, start, end):
+        ctx, environment_name, output_path, output_name, start, end, cameras):
```

Comment:
missing the `@click.option` above for camera.

Reply:
and also maybe using the arg somewhere?

Reply:
@optimuspaul  - Great catch (doh)! Changes pushed. Take one last quick look?